lib/rbac/filterer.rb

```diff
@@ -656,7 +656,9 @@ def apply_scope(klass, scope)
     end
 
     def apply_select(klass, scope, extra_cols)
-      scope.select(scope.select_values.blank? ? klass.arel_table[Arel.star] : nil).select(extra_cols)
```

Comment:
Per the commit message for this change:

> ####  Fix Rbac#apply_select extra_cols logic
> 
> Fixes and issue with `Rbac#apply_select` when `:extra_cols` exists and an
> already existing `.select` is in the scope.  Currently, this will throw
> an error (I guess).


Yes? (I honestly forget...)